validphys2/src/validphys/theorycovariance/construction.py

```diff
@@ -31,37 +31,6 @@
 
 log = logging.getLogger(__name__)
 
-theoryids_procs_central_values = collect(procs_central_values, ("theoryids",))
-
-theoryids_procs_central_values_no_table = collect(procs_central_values_no_table, ("theoryids",))
-
-collected_theoryids = collect("theoryids", ["theoryconfig"])
-
-
-def make_scale_var_covmat(predictions):
-    raise Exception(
-        "If you are seeing this error, please open an issue. This function should not be used"
-    )
-
-
-@check_correct_theory_combination
-def theory_covmat_singleprocess_no_table(
-    theoryids_procs_central_values_no_table, procs_index, theoryids, fivetheories
-):
-    """Calculates the theory covariance matrix for scale variations.
-    The matrix is a dataframe indexed by procs_index."""
-    s = make_scale_var_covmat(theoryids_procs_central_values_no_table)
-    df = pd.DataFrame(s, index=procs_index, columns=procs_index)
-    return df
-
-
-@table
-@check_correct_theory_combination
-def theory_covmat_singleprocess(theory_covmat_singleprocess_no_table, fivetheories):
```

Comment:
This function is still used in other places in this module, so either this function should not be removed, or all those function relying on this should (recursively) be removed as well